README.md

```diff
@@ -6,6 +6,14 @@
 
 This will provide you with a starting Drupal project that is managed with Composer. The install will include a small set of contrib modules, a starting custom module for specific for the build, and a custom starting theme generated from Emulsify.
 
+# Installation
+
+## Requirements
+Without these you will have difficulty installing this project.
+
+1. [PHP 7.4](http://www.php.net/)
+2. [Node ~12.13.0 \(we recommend NVM\)](https://github.com/creationix/nvm)
```

Comment:
@mikeethedude In reality we should be using Node 16 because that is what the latest version of Emulsify will require along with the Emulsify CLI.

Reply:
True. I was going off of the nvm setting which is lts/erbium or 12.13.0